workflows/differentialabundance.nf

```diff
@@ -357,49 +358,67 @@ workflow DIFFERENTIALABUNDANCE {
         ch_processed_matrices = ch_norm
             .map{ it.tail() }
             .first()
-    }
-    else{
 
-        DESEQ2_NORM (
-            ch_contrasts.first(),
-            ch_samples_and_matrix,
-            ch_control_features,
-            ch_transcript_lengths
-        )
+    } else {
+        if (params.differential_use_limma) {
 
-        // Run the DESeq differential module, which doesn't take the feature
-        // annotations
+            LIMMA_RNASEQ (
```

Comment:
This, and the versions bit below, is identical with the array limma usage- you can rearrange the logic to avoid the duplication